UnitySDK/Assets/ML-Agents/Scripts/Sensor/ISensor.cs

```diff
@@ -56,4 +56,24 @@ public interface ISensor {
         string GetName();
     }
 
+    public static class ISensorExtensions
+    {
+        /// <summary>
+        /// Get the total number of elements in the ISensor's observation (i.e. the product of the shape elements).
+        /// </summary>
+        /// <param name="sensor"></param>
+        /// <returns></returns>
+        public static int ObservationSize(this ISensor sensor)
```

Comment:
Needed this (product of the shape components) in a few places so added it as an extension to ISensor. Any objections?

Reply:
LGTM